### PR TITLE
Final macos10 notarised UDA installer builds for release ...

### PR DESCRIPTION
**Pull request by HughWilliams**

### CODE REVIEW

File: enterprise-uda-installer-archives-mapping.ttl [hunk 1]
```diff
@@ -11736,7 +11736,7 @@ source: a schema:CreativeWork ;
     a oplinst:FileSizeSpecification, schema:CreativeWork ;
     schema:name "File Size Details -- mxj3zzzz.dmg on Mac OS X 10.7.x (Universal) (with location Amazon S3 HTTP Depot)" ;
     oplinst:hasMeasurementUnit "Bytes" ;
-    oplinst:hasMeasurementValue "13231233"^^xsd:integer ;
+    oplinst:hasMeasurementValue "2307457"^^xsd:integer ;
```
Comment (TallTed) [inline enterprise-uda-installer-archives-mapping.ttl:11739]: The new archive is 11 MB smaller than previous?  That suggests to me that something's gone wrong, or inadvertently been left out, when building this archive.

**Review by TallTed — COMMENTED** [review]
Writeup in Community post says these new builds support 10.9+, but all these descriptions say 10.7+.  Which is correct?

**Comment by TallTed** [comment]
@openlink @HughWilliams -- I think this PR was merged prematurely.  See notes above.

File: enterprise-uda-installer-archives-mapping.ttl (hunk 1 above)
Comment (HughWilliams) [reply enterprise-uda-installer-archives-mapping.ttl:11739]: The size is correct as the previous (current live) installers was actually includes the JDBC-ODBC Lite Bridge Shared library components which should not have been included, so now they are removed such that it just includes the JAR file require for MT JDBC client connection ...

**Comment by HughWilliams** [comment]
Yes, should be 10.9+ so will update the descriptions and create new pull request ...

**Pull request merged**